lib/scrape.js

```diff
@@ -212,17 +246,20 @@ async function findWiki(name) {
       return {
         wikipedia: wikipediaurls,
         wikidata: wikidataurl,
+        stack_exchange,
```

Comment:
Please read this one <https://github.com/airbnb/javascript#objects--grouped-shorthand>

Reply:
Fixed, sorry for so many style errors.